redpanda/templates/configmap.yaml

```
@@ -39,30 +54,49 @@ data:
     organization: {{ .Values.organization }}
     {{- end }}
     redpanda:
-{{- if .Values.auth.sasl.enabled }}
-  {{- with .Values.auth.sasl.users }}
+{{- if not (include "redpanda.semver" . | semverCompare ">=22.1.1") }}
+  {{- if .Values.auth.sasl.enabled }}
+    {{- with .Values.auth.sasl.users }}
       superusers:
-    {{ toYaml . | indent 8 }}
+    {{ toYaml . | indent 4 }}
```

I believe L60-L61 should be the same as L38:
```suggestion
      superusers: ['{{ (first .).name }}']
```

---

Fixed in [60a4927](https://github.com/redpanda-data/helm-charts/pull/117/commits/60a4927e61db5d117d538345f8b246d1fca73a0b) - I think this was a throw back to the alternative list notation.